api/logging/logger.js

```diff
@@ -0,0 +1,15 @@
+import Winston from 'winston';
+
+const logger = Winston.createLogger({
```

Comment:
Create custom WinstonLogger class
steps:
1. create lib/logger directory
2. create winstonLogger.js file
3. move all that you have inside the api/logging/logger.js file to lib/logger/winstonLogger.js
4. create class "WinstonLogger" inside  lib/logger/winstonLogger.js
5. WinstonLogger should have info, warn, debug and report methods
6. Move Winston.createLogger to the constructor of the WinstonLogger class
7. You should be able to pass {name, args, error} into the WinstonLogger.report method
8. Create an entity of WinstonLogger and export it 
9. Use lib/logger/index.js to import your logger from lib/logger/winstonLogger.js
10. User logger.report() to log errors in your project

**Note** Please, import logger to server.js and other files from lib/logger/index.js but keep realization of your logger inside lib/logger/winstonLogger.js